backend/src/services/cloudinary/cloudinary.ts

```diff
@@ -0,0 +1,9 @@
+import cloudinary from 'cloudinary';
+
+cloudinary.v2.config({
+  cloud_name: process.env.CLOUDINARY_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_API_SECRET,
+});
```

Comment:
upload preset - `user_images` should be stored as a env variable

Reply:
This file could be named in better way -> as this is storing configuration, maybe `backend/src/services/cloudinary/config.ts`